api/test_api.py

```diff
@@ -96,3 +96,60 @@ def test_me_endpoint(mock_get_current_user):
     assert response.status_code == 200
     assert ('_id' and 'username' and 'hashed_password' and
             'active') in response.json()
+
+
+@pytest.fixture()
+def mock_init_sub_id(mocker):
+    async_mock = AsyncMock()
+    mocker.patch('api.pubsub.PubSub._init_sub_id',
+                 side_effect=async_mock)
+    return async_mock
+
+
+@pytest.fixture()
+def mock_listen(mocker):
+    async_mock = AsyncMock()
+    mocker.patch('api.pubsub.PubSub.listen',
+                 side_effect=async_mock)
+    return async_mock
+
+
+def test_listen_positive(mock_get_current_user, mock_init_sub_id, mock_listen):
+
+    user = User(username='bob',
+                hashed_password='$2b$12$CpJZx5ooxM11bCFXT76/z.o6HWs2sPJy4iP8.'
+                                'xCZGmM8jWXUXJZ4K',
+                active=True)
+    mock_get_current_user.return_value = user
+    mock_listen.return_value = 'String'
```

Comment:
I think we should return the more relevant string message for `mock_listen.return_value` instead of `String`. 
Something like what the actual /listen endpoint would return  `Listening for events on channel 1`

Reply:
I agree, I'd rather have something more informative there